### PR TITLE
Mark all as read/delete all | remove old notification

### PR DESCRIPTION
**Pull request by pistej**

### CODE REVIEW

**Comment by machour** [comment]
Nice work. I'll review this PR coming week and will merge it.

Thanks a lot !

**Comment by machour** [comment]
Hey @pistej, thanks again for this PR. Could you take my comments in account and re-submit your PR ?

**Comment by pistej** [comment]
Where can I find your comments ?

**Comment by machour** [comment]
Check the files changed tab:
https://github.com/machour/yii2-notifications/pull/39/files

**Comment by pistej** [comment]
I see nothing. Only my changes.

File: assets/notifications.js [hunk 1]
```diff
@@ -336,6 +362,66 @@ var Notifications = (function(options) {
         });
     };
 
+    /**
+     * Register click event on selector
+     */
+    this.registerClickEvents = function () {
+        if (self.opts.markAllSeenSelector !== null){
```
[the remark this answers is not in the document]
Comment (machour) [reply assets/notifications.js]: @pistej

**Comment by machour** [comment]
My bad, forgot to submit the review. You should be seeing them now.

**Comment by machour** [comment]
Merge, thanks a lot for your contribution !

**Pull request merged**